src/scss/objects/_form-controls.scss

```diff
@@ -0,0 +1,240 @@
+/**
+ * Custom Checkboxes and radios
+ *
+ * --------------------------------
+ *
+ * Checkbox button example HTML
+ *
+ *    <label class="c-form-label" for="checkbox1">
+ *        <input class="c-form-input" type="checkbox" id="checkbox1" name="myCheckbox" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--checkbox"></span>
+ *        My Checkbox Label Text
+ *    </label>
+ *
+ * -------------------------------
+ *
+ * Radio button example HTML
+ *
+ *    <label class="c-form-label" for="radio1">
+ *        <input class="c-form-input" type="radio" id="radio1" name="myRadioBtn" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--radio"></span>
+ *        My Radio Button Label Text
+ *     </label>
+ *
+ * ================================= */
+ $formControl-bgColor                  : $white !default;
+ $formControl-bgColor-disabled         : $color-disabled;
+ $formControl-borderColor              : $color-border;
+ $formControl-color             : $blue !default;
+ $formControl-background-color  : $color-bg !default;
+ $formControl-hover-color       : $blue--dark !default;
+ $formControl-borderColor-hover : $blue !default;
+ $formControl-margin-left       : 0 !default;
+ $formControl-padding-left      : spacing(x4) !default;
+ $formControl-width--wide       : 20px !default;
+ $formControl-width             : 24px !default;
+ $formControl-height--wide      : 20px !default;
+ $formControl-height            : 24px !default;
+ $formControl-border-width      : 1px;
+
+
+ .o-formControl-label {
+     position: relative;
+     display: block;
+     cursor: pointer;
+     user-select: none;
+     padding: spacing();
+     padding-left: $formControl-padding-left;
+
+     &:hover .o-formControl-indicator {
+         border-color: $formControl-borderColor-hover;
+     }
+
+     &:hover .o-formControl-input:disabled ~ .o-formControl-indicator {
+         border-color: $formControl-borderColor;
+         cursor: not-allowed;
+     }
+
+     .o-formControl-input:disabled ~ .o-formControl-indicator {
+        background-color: $formControl-bgColor-disabled;
+     }
+ }
+
+ // Give inline elements some space to the right
+ .o-formControl-label--inline {
+     display: inline-block;
+     padding-right: 2em;
+ }
+
+     // The hidden input
+     .o-formControl-input {
+         position: absolute;
+         opacity: 0;
+         z-index: -1; // Put the input behind the label so it doesn't overlay text
+
+         &:focus {
+             & + .o-formControl-indicator { @extend %u-elementFocus; }
+             & ~ .o-formControl-text { @extend %u-elementFocus; }
+         }
+
+         .no-csstransforms & {
+             position: relative !important;
+             opacity: 1;
+             z-index: 1;
```

Comment:
We should make use of the `$zIndex` map property here and anywhere else z-index is used, to keep them all consistent and in one place.

Reply:
@kevinrodrigues do you have an example of this as I've just ported this over from International.

Reply:
Something like `$zIndex(low)` should do it.

Reply:
@tommaton Just for reference, [this is the map](https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L113-L117) @kevinrodrigues is referring to.  If you need it to be specifically 1, then you can add a value to that map and reference that

Reply:
Cheers guys, I'll make that update